integration/monai/examples/spleen_ct_segmentation_local/job/app/config/config_fed_client.json

```diff
@@ -0,0 +1,68 @@
+{
+  "format_version": 2,
+
+  "executors": [
+    {
+      "tasks": [
+        "train", "submit_model", "validate"
+      ],
+      "executor": {
+        "id": "executor",
+        "path": "monai_nvflare.client_algo_executor.ClientAlgoExecutor",
+        "args": {
+          "client_algo_id": "client_algo",
+          "key_metric": "val_mean_dice"
+        }
+      }
+    }
+  ],
+
+  "task_result_filters": [
+  ],
+  "task_data_filters": [
+  ],
+
+  "components": [
+    {
+      "id": "client_algo",
+      "path": "monai.fl.client.MonaiAlgo",
+      "args": {
+        "bundle_root": "config/spleen_ct_segmentation",
+        "local_epochs": 10,
+        "tracking": {
+          "handlers_id": {
+              "trainer": {
+                  "id": "train#trainer",
+                  "handlers": "train#handlers"
+              },
+              "validator": {
+                  "id": "validate#evaluator",
+                  "handlers": "validate#handlers"
+              }
+          },
+          "configs": {
+              "tracking_uri": "http://127.0.0.1:5000",
+              "experiment_name": "monai_nvflare",
+              "is_not_rank0": "$torch.distributed.is_available() and torch.distributed.is_initialized() and torch.distributed.get_rank() > 0",
+              "trainer": {
+                  "_target_": "MLFlowHandler",
+                  "_disabled_": "@is_not_rank0",
+                  "tracking_uri": "@tracking_uri",
+                  "experiment_name": "@experiment_name",
+                  "iteration_log": true,
+                  "output_transform": "$monai.handlers.from_engine(['loss'], first=True)",
+                  "close_on_complete": true
+              },
+              "validator": {
+                  "_target_": "MLFlowHandler",
+                  "_disabled_": "@is_not_rank0",
+                  "tracking_uri": "@tracking_uri",
+                  "experiment_name": "@experiment_name",
+                  "iteration_log": false
+              }
+          }
+        }
```

Comment:
These are MONAI-specific config/parsing rules, not sure if put in the same file here in "config_fed_client.json" is better.
Or we have another config_monai.json or something that has these section and in MonaiAlgo class we read in the file?

Reply:
We don't support reading it from a file in this MONAI version, but this might be a good direction for future enhancement.

Reply:
This was simplified in the latest MONAI version. Now, we can use a config like this

```
  "tracking": "mlflow",
  "experiment_name": "monai_nvflare",
  "tracking_uri": "http://127.0.0.1:5000",

  "components": [
    {
      "id": "client_algo",
      "path": "monai.fl.client.MonaiAlgo",
      "args": {
        "bundle_root": "config/spleen_ct_segmentation",
        "local_epochs": 10,
        "train_args": {
            "tracking": "{tracking}",
            "tracking_uri": "{tracking_uri}",
            "experiment_name": "{experiment_name}"
        },
        "eval_args": {
            "tracking": "{tracking}",
            "tracking_uri": "{tracking_uri}",
            "experiment_name": "{experiment_name}"
        }
      }
    }
  ]
```

Reply:
Got it thanks good to know. Let's also updat this when the version is released